clang/test/dpct/driver-stream-and-event-enable-profiling.cu

```diff
@@ -0,0 +1,113 @@
+// RUN: dpct --enable-profiling  -out-root %T/driver-stream-and-event-enable-profiling %s --cuda-include-path="%cuda-path/include"
+// RUN: FileCheck --match-full-lines --input-file %T/driver-stream-and-event-enable-profiling/driver-stream-and-event-enable-profiling.dp.cpp %s
+
+#include<vector>
```

Comment:
Could you add check to make sure "#define DPCT_PROFILING_ENABLED" is define before device.hpp or dpct.hppis included?

Reply:
Okay